swap_meet/item.py

```diff
@@ -0,0 +1,23 @@
+class Item:
+    def __init__(self, category = ""):
+        # default parameter is what we want in case nothing is passed
+        self.category = category
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition == 5:
+            return "This fit is fire"
+        elif self.condition == 4:
```

Comment:
The condition of an item can have a decimal place.  What will happen if self.condition is 3.5?  How could you modify these conditionals to account for that possibility?  (PS: I love your descriptions!).